src/libs/dataUseTranslation.js

```diff
@@ -167,6 +167,32 @@ const consentTranslations = {
   }
 };
 
+export const getConsentTranslations = (restrictions) => {
+  let result = [];
+  if (restrictions.includes("NMDS")) {
+    result.push(consentTranslations.methodsResearch.description);
+  }
+  if (restrictions.includes("GSO")) {
+    result.push(consentTranslations.geneticStudiesOnly.description);
+  }
+  if (restrictions.includes("NPU")) {
+    result.push(consentTranslations.commercialUse.description);
+  }
+  if (restrictions.includes("PUB")) {
+    result.push(consentTranslations.publicationResults.description);
+  }
+  if (restrictions.includes("COL")) {
+    result.push(consentTranslations.collaboratorRequired.description);
+  }
+  if (restrictions.includes("IRB")) {
+    result.push(consentTranslations.ethicsApprovalRequired.description);
+  }
+  if (restrictions.includes("GS-")) {
+    result.push(consentTranslations.geographicalRestrictions.description);
+  }
+  return result;
+};
```

Comment:
I don't think we need this method. What we need to do is construct a valid `dataUse` and pass that into `translateDataUseRestrictions`. I should have caught this earlier, but didn't.

Reply:
okay, thank you!